### DISCUSSION TITLE
I can't use fabric.EraserBrush in fabric@4.x.x

### DISCUSSION THREAD

**Discussion by edthoms** [fabricjs/fabric.js#8268] [Q&A] [answered]
So I've been following examples of using the eraser brush, even [this one](https://codesandbox.io/s/focused-christian-7s5gx?file=/src/App.tsx) from the announcement of the feature, and I am trying to initialize `let erasingBrush = new fabric.EraserBrush(canvas)`
from the linked example and the docs on the website. But I get a console error telling me that `fabric.EraserBrush` is not a constructor.

I have checked the fabric module's files inside `node_modules` and I can see the `eraser_brush.mixin.js` is included in the file modules. But when analyzing the `fabric` object, `BaseBrush`, `CircleBrush`, `PencilBrush` appear, but no `EraserBrush`. 

I've tried to find a Custom Build to include Eraser to see if that fixes the problem, but it seems the only custom build that has that as a separate module is for version `5.2.0`, which I can't use because I'm relying on `4.x.x`. To clarify, I've tried using `fabric@4.5.1` and `fabric@4.6.0` to no avail.

What can I do? 

Thanks!

**Comment by ShaMan123** [comment, marked as answer]
You need to run a custom build as follows
```bash
cd ./node_modules/fabric
npm run build -- -x gestures accessors

```

This will create the files under `dist` with the eraser.
That example you referenced is outdated so be careful.

**Reply by edthoms** [reply]
Thank you, I want to add that the -x flag apparently was not enough to include erasing, maybe because I'm on linux. I instead modified the build script to include erasing module. 

From `"build": "node build.js modules=ALL requirejs exclude=gestures,accessors,erasing"` changed to 
`"build": "node build.js modules=ALL requirejs exclude=gestures,accessors"`

I know it won't persist through every update, but it seems to be a quick fix. 

Again, thanks!
